### PR TITLE
[CHI-362] Added calendar icon in CE date picker

### PR DESCRIPTION
**Pull request by janibolkvadze**
@jllr @mattnickles @dani-cl-madrid @SergioQCL 
    
https://ctl.atlassian.net/browse/CHI-362

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/262/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/263/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/265/ for more details.

File: src/custom-elements/src/components/date-picker/date-picker.tsx [hunk 1]
```diff
@@ -113,33 +113,38 @@ export class DatePicker {
...
+          placeholder={`mm/dd/yyyy`}
+          ref={el => (this._input = el as HTMLInputElement)}
           value={this.value}
-          format={this.format}
+          onChange={() => {
+            this.value = this._input.value;
+          }}
         />
-      </chi-popover>
+        <div class="a-icon sc-chi-icon -text--muted">
```
Comment (jllr) [inline src/custom-elements/src/components/date-picker/date-picker.tsx]: @janibolkvadze please use Icon custom element

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/291/ for more details.

**Pull request merged**